Source/WebCore/rendering/RenderElement.h

```diff
@@ -541,6 +543,13 @@ inline RenderObject* RenderElement::lastInFlowChild() const
     return nullptr;
 }
 
+inline bool RenderObject::shouldSkipContent() const
```

Comment:
There are too many functions with similar names, which makes this confusing.

1. RenderObject::shouldSkipContent
2. RenderStyle::determineEffectiveSkipsContent
3. RenderObject::isSkippedContent
4. RenderStyle::effectiveSkipContents

Can we change these to something like:

1. RenderObject::isSkippedContentRoot
2. isSkippedContentRoot(Element* element, RenderStyle& style) (see other comment)
3. RenderObject::isSkippedContent
4. RenderStyle::isSkippedContent


I think 1 & 2 should share names, ditto for 3 & 4, since they represent the same thing.

Reply:
I like shouldSkipContent as it is very clear, but it does state what should be done and not what it is. Given isInertSubtreeRoot exists, isSkippedContentRoot makes sense. Since it is a bit intrusive change I propose to do it in a separate PR (https://github.com/WebKit/WebKit/pull/15631).

Reply:
Regarding, the other half, I am not so sure. effectiveSkipContents is a bit of a strange flag, it gels set on the root and gets inherited (likely same as effectiveInert). However then isSkippedContent needs to test the fag is true, but on the parent, to filter out the root. So I am not sure we can really map them. Let me know what you think, it is easy to add the renaming to https://github.com/WebKit/WebKit/pull/15631, which likely should land first.

Reply:
I think `effectiveSkipContents` should be renamed to `effectiveSkippedContent` at least, because `SkipContents` implies it is the root skipped content (which is not true since the flag is inherited down)

Reply:
> I think `effectiveSkipContents` should be renamed to `effectiveSkippedContent` at least, because `SkipContents` implies it is the root skipped content (which is not true since the flag is inherited down)

That seems reasonable, I made https://github.com/WebKit/WebKit/pull/15640 for that rename.

Reply:
Renaming should now be done.